curriculum/challenges/english/01-responsive-web-design/applied-visual-design/adjust-the-size-of-a-header-versus-a-paragraph-tag.md

```diff
@@ -1,19 +1,19 @@
 ---
 id: 587d781b367417b2b2512abd
-title: Adjust the Size of a Header Versus a Paragraph Tag
+title: Adjust the Size of a Heading Tag Versus a Paragraph Tag
```

Comment:
We should not rename the title without renaming the file.

Also, this cannot be merged, until we have the redirects ready for backlinks.

Reply:
I have renamed the file name as well.